pipelines/commands/status_test.go

```diff
@@ -0,0 +1,140 @@
+package commands
+
+import (
+	"github.com/gookit/color"
+	"github.com/jfrog/jfrog-client-go/pipelines/services"
+	"testing"
+	"time"
+)
+
+func TestMonitorStatusChange(t *testing.T) {
+	type args struct {
+		pipeStatus string
+		reStatus   string
+	}
+	a1 := args{"waiting", "processing"}
+	a2 := args{"processing", "processing"}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{"test when result status is different from previous state", a1, true},
+		{"test when result status is same as previous", a2, false},
```

Comment:
Instead of using a1 + a2
```suggestion
		{"test when result status is different from previous state", args{"waiting", "processing"}, true},
		{"test when result status is same as previous", args{"processing", "processing"}, false},
```